packages/grid/data-grid/src/tests/selection.DataGrid.test.tsx

```diff
@@ -12,6 +12,49 @@ describe('<DataGrid /> - Selection', () => {
   // TODO v5: replace with createClientRender
   const render = createClientRenderStrictMode();
 
+  it('should be selected and unselected when clicking on a row', () => {
+    render(
+      <div style={{ width: 300, height: 300 }}>
+        <DataGrid
+          rows={[
+            {
+              id: 0,
+              brand: 'Nike',
+            },
+            {
+              id: 1,
+              brand: 'Adidas',
+            },
+          ]}
+          columns={[{ field: 'brand', width: 100 }]}
+        />
+      </div>,
+    );
+    const firstRow = getRow(0);
+    const secondRow = getRow(1);
+
+    // selecting row on click without key press
+    fireEvent.click(screen.getByRole('cell', { name: 'Nike' }));
```

Comment:
We have a helper to get the cell at X row and Y column.

```suggestion
    fireEvent.click(getCell(0, 0));
```